example.sync.yaml

```diff
@@ -60,6 +60,10 @@ filters:
   - name: DeclinedEvents
   # Events which cover the full day aren't synced
   - name: AllDayEvents
+  # Events which match the following regular expression, syntax here: https://github.com/google/re2/wiki/Syntax
```

Comment:
Please prefix this with "Exclude [events which...]". One particularly helpful prefix might be `(?i)` which enables case insensitive matching.